src/components/header/header-constants.tsx

```diff
@@ -0,0 +1,18 @@
+export const enum HeaderEnum {
+  default = 'default',
+  projects = 'projects',
+  joinUs = 'join-us',
+  about = 'about',
+}
+
+export const tabsData = [
```

Comment:
This should be in a file where the file name matches the variable.

Or you could put it into the component file.